app/code/community/Bolt/Boltpay/Model/Observer.php

```diff
@@ -298,6 +298,28 @@ public function setInitialOrderStatusAndDetails(Varien_Event_Observer $observer)
             ->save();
     }
 
+    /**
+     * Hides the Bolt Pre-auth order states from the admin->Sales->Order list
+     *
+     * event: sales_order_grid_collection_load_before
+     *
+     * @param Varien_Event_Observer $observer Observer event contains an orderGridCollection object
+     */
+    public function filterPreAuthOrders($observer) {
```

Comment:
let's call this `hidePreAuthOrders` ? 'filter' can be intepreted either way - either you filter (hide) or filter (select).

Reply:
done